### ISSUE TITLE
Generated rbi files for some gems don't properly check with latest sorbet

### ISSUE THREAD

**Issue by robinbrandt** [Shopify/tapioca#95]
Some rbi files that are generated by tapioca aren't properly type checked by Sorbet 0.5.5843.

Example for thor:

Gemfile:
```
# frozen_string_literal: true

source "https://rubygems.org"

git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }

gem "sorbet"
gem "tapioca"
gem "thor"
```

Gemfile.lock:
```
GEM
  remote: https://rubygems.org/
  specs:
    coderay (1.1.3)
    method_source (1.0.0)
    pry (0.13.1)
      coderay (~> 1.1)
      method_source (~> 1.0)
    sorbet (0.5.5843)
      sorbet-static (= 0.5.5843)
    sorbet-runtime (0.5.5843)
    sorbet-static (0.5.5843-universal-darwin-14)
    tapioca (0.4.0)
      pry (>= 0.12.2)
      sorbet-runtime
      sorbet-static (>= 0.4.4471)
      thor (>= 0.19.2)
    thor (1.0.1)

PLATFORMS
  ruby

DEPENDENCIES
  sorbet
  tapioca
  thor

BUNDLED WITH
   2.1.2
```

How to reproduce:

```shell
$ bundle exec srb init
$ bundle exec tapioca gen thor
$ bundle exec srb tc
```

Expected: No errors.

Got: 
```
sorbet/rbi/gems/thor@1.0.1.rbi:31: circular argument reference _ https://srb.help/2001
    31 |  def self.package_name(name, _ = _); end
```

Looks like that Sorbet version doesn't like the `_ = _` pattern for unused parameters. If you manually edit the rbi file an change this to `_unused = _` it works.

**Comment by Morriar** [comment]
I just pushed this https://github.com/Shopify/tapioca/pull/96 that should solve this problem.

**Comment by paracycle** [comment]
The fix for this was released in versions 0.4.1 and above, so closing this issue.
